### PR TITLE
Add sandwich video to landing page

### PR DESCRIPTION
**Pull request by fragosti**
## Description

https://dogfood.0xproject.com/

## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/website/ts/components/sections/landing/hero.tsx [hunk 1]
```diff
@@ -13,26 +15,68 @@ import { WebsitePaths } from 'ts/types';
...
+                    isFullWidth={true}
+                    description="0x is an open protocol that enables the peer-to-peer exchange of assets on the Ethereum blockchain."
+                    figure={<LandingAnimation image={<HeroAnimation />} />}
+                    actions={<HeroActions onPlayVideoClick={this._openModalVideo} />}
+                    // announcement={announcement}
+                />
+                <ModalVideo
+                    channel="youtube"
+                    isOpen={isVideoOpen}
+                    videoId="c04eIt3FQ5I"
```
Comment (dave4506) [inline packages/website/ts/components/sections/landing/hero.tsx]: should we put the videoId in constants?

File: packages/website/ts/components/sections/landing/hero.tsx [hunk 2]
```diff
@@ -13,26 +15,68 @@ import { WebsitePaths } from 'ts/types';
...
+                    onClose={this._closeModalVideo}
+                    youtube={{
+                        autoplay: 1,
+                        controls: 0,
+                        showinfo: 0,
+                        modestbranding: 1,
```
Comment (dave4506) [inline packages/website/ts/components/sections/landing/hero.tsx]: maybe put the video prop settings in constants? fine with either

File: packages/website/ts/components/sections/landing/hero.tsx (hunk 1 above)
Comment (fragosti) [reply packages/website/ts/components/sections/landing/hero.tsx]: Our rule is if it's used twice then it must be a variable, so I think we're good here.

File: packages/website/ts/components/sections/landing/hero.tsx (hunk 2 above)
Comment (fragosti) [reply packages/website/ts/components/sections/landing/hero.tsx]: I think it's good here.

**Comment by chriskalani** [comment]
Looks pretty good! Can we make it bigger though? It's pretty small.

**Comment by chriskalani** [comment]
Also, can we hid everything on top of the video when it first loads? These overlays require the user to mouse over and off to go away.
Screen Shot 2019-07-31 at 8 52 38 PM

**Pull request merged**